### DISCUSSION TITLE
How to use gpt-3.5？

### DISCUSSION THREAD

**Discussion by nuonuona** [srcnalt/OpenAI-Unity#24] [Q&A] [answered]
OpenAI更新了新的模型，我应该怎么用到Unity工程里边呢?

**Comment by srcnalt** [comment]
Hi @nuonuona I will overview the documentation and bring support for ChatGPT soon.

**Comment by srcnalt** [comment, marked as answer]
Just released the latest updates, CreateChatCompletion endpoint covers this now.

**Reply by ImabYa** [reply]
How can I chang the url, when I set the parm model to "gpt-3.5-turbo",unity debugs error:Error Message: This is a chat model and not supported in the v1/completions endpoint. Did you mean to use v1/chat/completions?
Error Type: invalid_request_error

**Reply by ImabYa** [reply]
I want to change model,but I have to change url. In this project ,there seems no way to change url. I need your help

**Reply by srcnalt** [reply]
You will need to use `CreateChatCompletion` method.
